Makefile

```diff
@@ -174,38 +182,47 @@ $(OBJECTS): $(OUTPUT)include/distortos/distortosConfiguration.h
 $(GENERATED): Makefile
 
 $(OUTPUT)%.o: %.S
-	$(AS) $(ASFLAGS) $(ASFLAGS_$(<)) -c $< -o $@
+	@echo " AS\r\t" $< 
+	$(Q)$(AS) $(ASFLAGS) $(ASFLAGS_$(<)) -c $< -o $@
 
 $(OUTPUT)%.o: %.c
-	$(CC) $(CFLAGS) $(CFLAGS_$(<)) -c $< -o $@
+	@echo " CC\r\t" $<
+	$(Q)$(CC) $(CFLAGS) $(CFLAGS_$(<)) -c $< -o $@
 
 $(OUTPUT)%.o: %.cpp
-	$(CXX) $(CXXFLAGS) $(CXXFLAGS_$(<)) -c $< -o $@
+	@echo " CXX\r\t" $<
+	$(Q)$(CXX) $(CXXFLAGS) $(CXXFLAGS_$(<)) -c $< -o $@
 
 $(OUTPUT)%.a:
-	rm -f $@
-	$(AR) rcs $@ $(filter %.o,$(^))
+	$(Q)rm -f $@
+	@echo " AR\r\t" $@
+	$(Q)$(AR) rcs $@ $(filter %.o,$(^))
 
 $(OUTPUT)%.elf:
+	@echo " LD\r\t" $@
 	$(eval ARCHIVES_$@ := -Wl,--whole-archive $(addprefix -l:,$(filter %.a,$(^))) -Wl,--no-whole-archive)
-	$(LD) $(LDFLAGS) -T$(filter %.ld,$(^)) $(filter %.o,$(^)) $(ARCHIVES_$(@)) -o $@
+	$(Q)$(LD) $(LDFLAGS) -T$(filter %.ld,$(^)) $(filter %.o,$(^)) $(ARCHIVES_$(@)) -o $@
 
 $(OUTPUT)%.hex:
-	$(OBJCOPY) -O ihex $(filter %.elf,$(^)) $@
+	@echo " OBJCOPY\r\t" $@
+	$(Q)$(OBJCOPY) -O ihex $(filter %.elf,$(^)) $@
 
 $(OUTPUT)%.bin:
-	$(OBJCOPY) -O binary $(filter %.elf,$(^)) $@
+	@echo " OBJCOPY\r\t" $@
+	$(Q)$(OBJCOPY) -O binary $(filter %.elf,$(^)) $@
 
 $(OUTPUT)%.dmp:
-	$(OBJDUMP) -x --syms --demangle $(filter %.elf,$(^)) > $@
+	@echo " OBJDUMP\r\t" $@
+	$(Q)$(OBJDUMP) -x --syms --demangle $(filter %.elf,$(^)) > $@
 
 $(OUTPUT)%.lss:
-	$(OBJDUMP) --demangle -S $(filter %.elf,$(^)) > $@
+	@echo " OBJDUMP\r\t" $@
+	$(Q)$(OBJDUMP) --demangle -S $(filter %.elf,$(^)) > $@
 
 .PHONY: size
 size:
-	$(SIZE) -B $(filter %.elf,$(^))
+	$(Q)$(SIZE) -B $(filter %.elf,$(^))
 
 .PHONY: clean
 clean:
-	$(RM) -r $(OUTPUT)
+	$(Q)$(RM) -r $(OUTPUT)
```

Comment:
I think this particular command should not be silenced, because otherwise there will be completely no output for "make clean"...